internal/broker/bind_create.go

```diff
@@ -123,6 +123,7 @@ func (b *BindEndpoint) Bind(ctx context.Context, instanceID, bindingID string, d
 		InstanceID: instanceID,
 
 		CreatedAt: time.Now(),
+		ExpiresAt: time.Now().Add(time.Duration(int64(expirationSeconds)) * time.Second),
```

Comment:
Are we sure the time of creation of this object is the same as the time of creation of the token?

Reply:
I'm not sure, I will update this to check the time returned in the token.